transcom/mymove
# INT B-20946 SSW max obligation calculation correction

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20946)

## Summary

Seems like the SSW packet was showing the wrong value for the 100% GCC under the weight & entitlements section. This was changed from `TotalWeight` to `Entitlement` since they don't want Pro Gear included in there.

Also removed old commented out code.

### How to test

1. You can use pre-existing test data or create your own (I created my own because some older test data does not include weight tickets which will trigger an error)
2. There have been some DTOD issues, so go ahead and make `DTOD_USE_MOCK=true` in your `envrc`
3. Hit up `direnv allow` and fire up your `server` and `client`
4. Set up a new customer with a PPM move, set up your shipment - request an advance - and sign
5. Go in as a SC, approve the advance and submit the move
6. Log back in as the customer and download your AOA packet - you should now see the `100% GCC` value show the ENTITLEMENT poundage instead of the estimated weight
![Screenshot 2024-08-27 at 10 16 54 AM](https://github.com/user-attachments/assets/f62b9a0a-c939-4586-9708-ba5928eb5f0c)
7. Go ahead and complete your PPM by uploading your weight tickets
8. Log back in as a SC and close it out by reviewing the weight tickets and approving everything
9. You can review the AOA packet and the Payment Packet as the office user to confirm both contain the same change
![Screenshot 2024-08-27 at 10 21 32 AM](https://github.com/user-attachments/assets/56dd06e1-bbcc-4b95-bce3-1566b373e6ba)
![Screenshot 2024-08-27 at 10 21 52 AM](https://github.com/user-attachments/assets/05c033d3-566b-457f-8618-3838b3586591)
